MobileCenterUpdates/MobileCenterUpdates/MSUpdates.m

```diff
@@ -335,8 +340,8 @@ - (void)handleUpdate:(MSReleaseDetails *)details {
                MS_DEVICE.systemVersion);
     return;
   }
-
```

Comment:
I mean you are removing an empty line above the comment and adding it below the comment.